tree/ntuple/v7/src/RColumnElement.hxx

```diff
@@ -267,6 +267,10 @@ namespace {
 using ROOT::Experimental::EColumnType;
 using ROOT::Experimental::Internal::RColumnElementBase;
 
+// testing value for an unknown future column type
+inline constexpr EColumnType kTestFutureType =
```

Comment:
It is in a public header and thus part of the public interface.  If we can not move it to a private header, can we put it into the `Internal` namespace?

Reply:
This header is not public, it's in the `src` directory and only included by our cxx files.